obspy/io/seiscomp/inventory.py

```diff
@@ -374,9 +431,12 @@ def _read_channel(inventory_root, cha_element, _ns):
         response_id = sensor_element.get("response")
         response_elements = []
 
-        for resp_type in ['responsePAZ', 'responsePolynomial']:
-            search = "{}[@publicID='{}']".format(resp_type, response_id)
-            response_elements += inventory_root.findall(_ns(search))
+        for resp_type in instrumentation_register["responses"].keys():
```

Comment:
Doesn't seem like you need `resp_type`.
```suggestion
        for resp_elements in instrumentation_register["responses"].values():
```

Reply:
Changed as suggested.